web/modules/custom/dpl_patron_menu/dpl_patron_menu.module

```diff
@@ -31,3 +33,57 @@ function dpl_patron_menu_preprocess_page(array &$variables): void {
     'menu' => $plugin_block->build(),
   ];
 }
+
+/**
+ * Implements hook_form_FORM_ID_alter().
+ *
+ * @param array<mixed> $form
+ *   An associative array containing the structure of the form.
+ * @param \Drupal\Core\Form\FormStateInterface $form_state
+ *   The current state of the form.
+ */
+function dpl_patron_menu_form_dpl_library_agency_list_size_settings_form_alter(array &$form, FormStateInterface $form_state): void {
+  $config = \Drupal::service('config.factory')
+    ->getEditable('dpl_library_agency.list_size_settings');
```

Comment:
You should use config service here instead.

Reply:
Are now using config service instead of config factory.